packages/shared/src/lib/auxiliary/wallet-connect/actions/updateAccountForDappSession.ts

```diff
@@ -13,6 +13,13 @@ export async function updateAccountForDappSession(dappSession: ISession, account
 
     const protocols = Object.keys(dappSession.namespaces)
     for (const protocol of protocols) {
+        if (
+            !dappSession.namespaces[protocol] ||
```

Comment:
is it worth a helper function to see if the method is supported? and can be used in multiple places before we do anything